### DISCUSSION TITLE
WMS support for single tile?

### DISCUSSION THREAD

**Discussion by jatech1** [Mapsui/Mapsui#1989] [Q&A]
Trying to determine if the WMS layer supports single tile (ie. where the request made to WMS server is for single full map image and not individual tiled pieces of it).  If not, can you point me to any code I can override to add this support?

**Comment by pauldendulk** [comment]
@jatech1 Yes, that is the more regular behavior. The tiles request of a WMS is to optimize performance. There is a sample for this in the file 'WmsSample.cs'. However, I just noticed that the service can not be reached. The code itself might be useful however.
